server/utils.go

```diff
@@ -223,3 +223,18 @@ func (s *Server) FilterDisallowedAnnotations(toFind, toFilter map[string]string,
 
 	return s.config.Workloads.FilterDisallowedAnnotations(allowed, toFilter)
 }
+
+// stopTimeoutFromContext returns the stop timeout in seconds for the provided
+// context. If the context has no timeout or deadline set, then it will default
+// to 10s.
+func stopTimeoutFromContext(ctx context.Context) int64 {
+	// Default timeout in seconds
+	timeout := int64(10)
```

Comment:
Nit: we could move this to a variable atop the file to make it an easy to find default.